CHANGELOG.md

```diff
@@ -2,30 +2,37 @@
 
 _released `2023-03-21`_
 
-Version `14.0.0` is a breaking change release that contains some backwards-incompatible changes that may require you to provide additional meta information about your app (if that information isn't already provided). If you are missing this information, you will be blocked from making major updates to your application (`zapier promote`, `zapier upload`, `zapier push`).
+Version `14.0.0` is a breaking change release that contains several important upgrades and deprecations. Here is a brief breakdown of the most notable changes (see further below for a more detailed list of changes):
+
+1. `alters_custom_fields` is no longer set to `true` all the time. Developers should now set the flag appropriately. This will signal to the editor when needs have to be re-fetched (i.e. user picked another spreadsheet so we need to refresh the dropdowns that act on column/rows).
```

Comment:
`needs` is not a public term, we should say `dynamic dropdown choices` or similar instead

Reply:
FWIW, when a dynamic dropdown `altersDynamicFields`, it's to refetch other input fields, not the dynamic dropdown choices themselves. I've suggested a change on the copy.